gradle/libs.versions.toml

```diff
@@ -35,9 +35,9 @@ coroutines-test = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-test", ve
 
 google-accompanist-swipeRefresh = { module = "com.google.accompanist:accompanist-swiperefresh", version = "0.20.3" }
 
-gradlePlugin-android = { module = "com.android.tools.build:gradle", version = "7.0.3" }
+gradlePlugin-android = "com.android.tools.build:gradle:7.1.0"
```

Comment:
Intellij doesn't support this AGP version yet, so we should stick to the 7.0.x line for now.

Reply:
ok wasn't aware that we also wait for Intellij compatibility before upgrading

Reply:
We relaxed that for a while last summer when we first added Compose because it was AS-only when it went 1.0. But more recently we've been able to support both again.